### PR TITLE
Add DRS form letters

### PR DESCRIPTION
**Pull request by billyhunt**
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1129)
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1130)
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1131)
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1132)

## What does this change?

We are adding 4 DRS form letters to refer users to the correct agency.  I added all of these in the same PR to minimize the chance  of problems in the database migration.   

## Screenshots (for front-end PR):

image
image
image
image
image

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/migrations/0126_drs_hhs_referral_letter.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+U.S. Department of Justice
+Civil Rights Division
+""")
+
+
+def remove_drs_hhs_referral_letter(apps, schema_editor):
+    ResponseTemplate = apps.get_model('cts_forms', 'ResponseTemplate')
+    templates = ResponseTemplate.objects.filter(title__icontains='DRS - EEOC Referral Form Letter')
```
Comment (louh) [inline crt_portal/cts_forms/migrations/0126_drs_hhs_referral_letter.py:38]: I think the filter string here needs to match this template name

File: crt_portal/cts_forms/migrations/0127_drs_dot_referral_letter.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+U.S. Department of Justice
+Civil Rights Division
+""")
+
+
+def remove_drs_dot_referral_letter(apps, schema_editor):
+    ResponseTemplate = apps.get_model('cts_forms', 'ResponseTemplate')
+    templates = ResponseTemplate.objects.filter(title__icontains='DRS - EEOC Referral Form Letter')
```
Comment (louh) [inline crt_portal/cts_forms/migrations/0127_drs_dot_referral_letter.py:38]: Same comment re: filter string

File: crt_portal/cts_forms/migrations/0128_drs_dept_of_ed_referral_letter.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+U.S. Department of Justice
+Civil Rights Division
+""")
+
+
+def remove_drs_dept_of_ed_referral_letter(apps, schema_editor):
+    ResponseTemplate = apps.get_model('cts_forms', 'ResponseTemplate')
+    templates = ResponseTemplate.objects.filter(title__icontains='DRS - EEOC Referral Form Letter')
```
Comment (louh) [inline crt_portal/cts_forms/migrations/0128_drs_dept_of_ed_referral_letter.py:39]: Same comment re: filter string

**Review by louh — CHANGES_REQUESTED** [review]
@billyhunt Can you also squash/edit/rebase your commit history to remove the churn on the address template?

**Comment by billyhunt** [comment]
> @billyhunt Can you also squash/edit/rebase your commit history to remove the churn on the address template?

Lets chat about it.  I don't usually like to modify or squash commit histories, but am open to changing.

**Review by louh — APPROVED** [review]

**Pull request merged**